docs/getting_started/install_ez_platform.md

```diff
@@ -24,13 +24,19 @@ Before getting started, make sure you review the [requirements](requirements.md)
 
 ## Get Composer
 
-Install Composer, the PHP command line dependency manager.
+Install the recent stable version of Composer, the PHP command line dependency manager.
 Use the package manager for your Linux distribution. For example on Ubuntu:
 
 ``` bash
 apt-get install composer
 ```
 
+To verify you got a recent stable version of Composer, run:
```

Comment:
Maybe place it in a tip box? It's not an obligatory step

Reply:
We would end up having two tip boxes one after another — if visually it's okay, then sure! :)

Reply:
Fair enough. Or you could change it to `you can run:` to emphasise that the action is optional.

Reply:
Sure, no problem!